### ISSUE TITLE
Basic Plasma Turbine collider and model issue

### ISSUE THREAD

**Issue by Powermonkey00** [RavienGaming/Reforged-Eden-2#204]
### Reforged Eden 2 Build Number

15

### Game mode

Creative

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

Small plasma turbine can be selected/shot through adjacent diagonal blocks at the upper corners.  Model passes through adjacent diagonal wedge blocks at lower corners.

**edit: just discovered this is actually true of several other things.  The issue may be the diagonal steel blocks.

Selecting the turbine through steel blocks:
![image](https://github.com/RavienGaming/Reforged-Eden-2/assets/51749577/8ecced61-7c40-411d-b118-4c5b45647e63)

Bottom of model peeking through steel blocks
![image](https://github.com/RavienGaming/Reforged-Eden-2/assets/51749577/40f06626-7f8c-4b92-a137-c32d65ddbd5a)

![image](https://github.com/RavienGaming/Reforged-Eden-2/assets/51749577/7ee96807-4b14-4c18-a69b-84b67b02326e)

**edit: Selecting a CPU extender through the diagonal steel blocks

**Comment by VermillionX** [comment]
Yeah, that'll happen on irregularly shaped deco blocks vs extremely narrow armor blocks. Colliders go over the deco model, which means their edges will clip through the narrowest colliders on other blocks.
